hystrix/eventstream.go

```diff
@@ -162,13 +165,13 @@ func (sh *StreamHandler) publishThreadPools(pool *executorPool) error {
 		CurrentMaximumPoolSize: uint32(pool.Max),
 
 		RollingStatsWindow:          10000,
-		QueueSizeRejectionThreshold: 0,
-		CurrentQueueSize:            0,
+		QueueSizeRejectionThreshold: uint32(pool.QueueSizeRejectionThreshold),
+		CurrentQueueSize:            uint32(pool.WaitingCount()),
 	})
 	if err != nil {
 		return err
 	}
-	err = sh.writeToRequests(eventBytes)
+	_ = sh.writeToRequests(eventBytes)
```

Comment:
Should probably `return sh.writeToRequests(eventBytes)` instead of ignoring the error.

Reply:
fixed